met-api/src/met_api/services/widget_timeline_service.py

```diff
@@ -56,13 +86,13 @@ def _create_timeline_model(widget_id: int, timeline_data: dict):
         for event in timeline_data.get('events', []):
             timeline_model.events.append(
                 TimelineEventModel(
-                    widget_id = widget_id,
-                    engagement_id = event.get('engagement_id'),
-                    timeline_id = event.get('timeline_id'),
-                    description = event.get('description'),
-                    time = event.get('time'),
-                    position = event.get('position'),
-                    status = event.get('status'),
+                    widget_id=widget_id,
```

Comment:
Did the linter ask for the removal of these spaces, is there a rule associated with this relevant to classes? I personally prefer when they're spaced out - to me that's more readable.